kde-apps/krdc/patches/krdc-24.02.2.patchset

```diff
@@ -0,0 +1,91 @@
+From 41e8b29c458bef3a0dbc791babca5a2e5406f57f Mon Sep 17 00:00:00 2001
+From: Sergei Reznikov <diver@gelios.net>
+Date: Fri, 24 Aug 2018 17:55:51 +0300
+Subject: Fix build on Haiku
+
+
+diff --git a/cmake/modules/FindLibSSH.cmake b/cmake/modules/FindLibSSH.cmake
+index c1fecfd..1426449 100644
+--- a/cmake/modules/FindLibSSH.cmake
++++ b/cmake/modules/FindLibSSH.cmake
+@@ -17,6 +17,8 @@ find_path(LIBSSH_INCLUDE_DIR
+     /usr/local/include
+     /opt/local/include
+     /sw/include
++	/system/develop/headers
++	/system/develop/headers/x86
+     ${CMAKE_INCLUDE_PATH}
+     ${CMAKE_INSTALL_PREFIX}/include
+ )
+@@ -30,6 +32,8 @@ find_library(SSH_LIBRARY
+     /usr/local/lib
+     /opt/local/lib
+     /sw/lib
++	/system/develop/headers
++	/system/develop/headers/x86
+     ${CMAKE_LIBRARY_PATH}
+     ${CMAKE_INSTALL_PREFIX}/lib
+ )
+diff --git a/vnc/CMakeLists.txt b/vnc/CMakeLists.txt
+index 20f74f0..6dd2933 100644
+--- a/vnc/CMakeLists.txt
++++ b/vnc/CMakeLists.txt
+@@ -31,6 +31,10 @@ target_link_libraries(krdc_vncplugin
+     krdccore
+ )
+ 
++if(HAIKU)
++    target_link_libraries(krdc_vncplugin network)
++endif ()
++
+ if(QT_MAJOR_VERSION STREQUAL "6")
+     target_link_libraries(krdc_vncplugin KF6::KCMUtils)
+ endif()
+diff --git a/vnc/vncclientthread.cpp b/vnc/vncclientthread.cpp
+index 5b5804c..9a5ce10 100644
+--- a/vnc/vncclientthread.cpp
++++ b/vnc/vncclientthread.cpp
+@@ -667,6 +667,7 @@ void VncClientThread::clientSetKeepalive()
+         return;
+     }
+ 
++#ifdef TCP_KEEPIDLE
+     optval = m_keepalive.intervalSeconds;
+     if (setsockopt(cl->sock, IPPROTO_TCP, TCP_KEEPIDLE, &optval, optlen) < 0) {
+         qCritical(KRDC) << "setsockopt(TCP_KEEPIDLE)" << strerror(errno);
+@@ -684,6 +685,7 @@ void VncClientThread::clientSetKeepalive()
+         qCritical(KRDC) << "setsockopt(TCP_KEEPCNT)" << strerror(errno);
+         return;
+     }
++#endif
```

Comment:
#endif should come two lines later

Reply:
Ah, kept it in place as it was done prior, will fix. :)

Reply:
Should be good now, this if for the whole block "m_keepalive" then?

Reply:
good to me